### DISCUSSION TITLE
Upcoming unlockables change

### DISCUSSION THREAD

**Discussion by Leroy8888** [HelixNebulaStudio/RiseOfTheDead#260] [General]
Starting off, I do think this idea is fine. 

However I had some concerns to how the unlocked ones would be handled if the update was released. Not everyone has an unlockable on every clothing item let alone each individual item and it can be quite difficult to get 4 top hat before the update launches. 

I think the consumed unlockables should be refunded to the player upon launch so they can use them on clothing items they please or maybe even sell them. @MrGarrettFerret suggested the idea of doing this via the mailbox feature. Transforming unlocked cosmetics into claimable rewards from there.

If something is already in place to avoid this concern then I suppose this post can be closed.

**Comment by MXKhronos** [comment]
All existing clothing will have your previously unlocked skins on each of your items. For example, if you `/loadmainsave`, all `gasmask` in your storages will have all your gasmask unlockables applied to them. Any newly obtained gasmask will not however.

You will be able to freely switch between each skin on the workbench same as before. The difference is you won't be able to apply rare skins on new items for free and then selling them devaluing the rarity.

**Comment by Leroy8888** [comment]
Alright, but what will happen to unlocked skins of which you do not have a clothing piece in your storage? 

For example, there probably are a few players who do not have a dufflebag anymore since they now use a traveler's backpack. Would they just lose all the unlockables tied to the dufflebag, despite having all or some unlockables for it?

And as for the 2nd paragraph, what exactly do you mean with ''You won't be able to apply rare skins on new items for free and then sell them to devalue the rarity?''. Do you have to pay a small gold fee in order to apply unlockable patterns after the update? Or does this talk about the fact that you will only be able to use an unlockable once on selected clothing piece, thus avoiding devaluation of the unlockable?

**Reply by MXKhronos** [reply]
Oh good point, I'll work out a charge system then so you'll still have those skins even if you do not have the clothing in your storages. E.g. For each pre unlocked skins, you will get 1 additional charges which you can apply using the workbench. This is on top of having all your existing clothing with your pre unlocked skins. So that if you have unlocked skins on e.g. skullmask and do not have it atm, you can still apply the skin on a new skull mask while consuming a charge. It will no consume a charge if your skullmask already has the skin unlocked.

Oh, by "for free", I just mean you can't create a bunch of inferno `vexgloves` on the workbench just by setting the appearance because now, one skin perm unlocked one inferno `vexgloves`. Using skin perm will be free.
